src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -0,0 +1,68 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import java.util.function.BooleanSupplier;
+import java.util.function.DoubleSupplier;
+
+@Logged
+public class Elevator extends SubsystemBase {
+
+    private static final double POSITION_THRESHOLD = Units.inchesToMeters(1);
+
+    private final ElevatorIO io;
+    private final ElevatorInputs inputs = new ElevatorInputs();
+
+    private double targetPosition = 0.0;
+
+    public Elevator(ElevatorIO io) {
+        this.io = io;
+    }
+
+    @Override
+    public void periodic() {
+        io.updateInputs(inputs);
+    }
+
+    public double getVelocityMotor() {
```

Comment:
Remove the `Motor` suffix. Users of the elevator subsystem may be confused and think that the return value will be the velocity of the motor in motor rotations per second vs the velocity of the elevator in meters per second.